<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QTextEdit" name="str1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36,36,36);
color: white;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="str2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36,36,36);
color: white;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Антиплагиатор 3000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(36,36,36);
color: white;
border-radius: 10px;
}
QPushButton:hover {
  background-color: rgb(100,100,100);
color: white;
border-radius: 10px;
}
QPushButton:pressed {
   background-color: rgb(50,50,50);
color: white;
border-radius: 10px;
}
QPushButton:disabled {
   background-color: rgb(70,70,70);
color: white;
border-radius: 10px;
}

</string>
   </property>
   <property name="text">
    <string>Сверить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Укажи порог ругательства</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Porog">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36,36,36);
color: white;
border-radius: 5px;</string>
   </property>
   <property name="prefix">
    <string comment="%" extracomment="%"/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>881</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
